app/controllers/users_controller.rb

```diff
@@ -44,14 +45,36 @@ def destroy
     redirect_to user_url
   end
 
+  def following 
+    @title = "Following"
+    @user = User.find(params[:id])
+    @user = @user.followed_users.paginate(page: params[:pate], per_page: 5)
+    @all_users = @user.followed_users
+    render 'show_follow'
+  end
+
+  def followers
+    @title ="Followers"
+    @user = User.find(params[:id])
+    @user = @user.followers_users.paginate(page: params[:pate], per_page: 5)
+    @all_users = @user.followers_users
```

Comment:
1. .followers_user should be **.followers**
2. @user should be plural
3. :page mispelling 
```suggestion
    @users = @user.followers.paginate(page: params[:page], per_page: 5)
    @all_users = @user.followers
```
Observe the changes I made.